src/database/DataTypes/Item.js

```diff
@@ -272,6 +272,16 @@ export class Item extends Realm.Object {
 
     return totalDosesPossible;
   }
+
+  /**
+   * Updates name and linked requisition `itemName` data. Finalised requisitions are not updated.
+   */
+  updateName(name) {
+    this.name = name;
+    this.requisitionItems.filtered("requisition.status != 'finalised'").forEach(requisitionLine => {
+      requisitionLine.itemName = name;
+    });
```

Comment:
I can't remember if it's good practice to use curlies for `forEach` one-liners... probably not, but eslint didn't complain.

Reply:
Will need to `UIDatabase.save()` the `RequisitionItem` records so that they are re-synced.

Reply:
Missing a paramater on the change: https://github.com/sussol/react-native-database/blob/81e18edcfe935d796412fa8ffcae6ba69c2a4e94/src/Database.js#L163